launchable/commands/compare/subsets.py

```diff
@@ -0,0 +1,46 @@
+import click
+from tabulate import tabulate
+
+
+@click.command()
+@click.argument('file_before', type=click.Path(exists=True))
+@click.argument('file_after', type=click.Path(exists=True))
+def subsets(file_before, file_after):
+    """
+    Compare two subset files and display changes in test order positions.
+    """
+
+    # Read files and map test paths to their indices
+    with open(file_before, 'r') as f:
+        before_tests = f.read().splitlines()
+    before_index_map = {test: idx for idx, test in enumerate(before_tests)}
+
+    with open(file_after, 'r') as f:
+        after_tests = f.read().splitlines()
+    after_index_map = {test: idx for idx, test in enumerate(after_tests)}
+
+    changes = []
```

Comment:
Can you define the type for this variable?

e.g. https://github.com/cloudbees-oss/smart-tests-cli/blob/e1ff1d31edee7a6f67649d4e00edb5b0ec59f94a/launchable/utils/http_client.py#L24-L25